lib/matplotlib/tests/test_ticker.py

```diff
@@ -228,6 +228,17 @@ class TestScalarFormatter(object):
 
     use_offset_data = [True, False]
 
+    scilimits_data = [
+        (False, (0, 0), (10.0, 20.0), 0),
+        (True, (-2, 2), (-10, 20), 0),
+        (True, (-2, 2), (-20, 10), 0),
+        (True, (-2, 2), (-110, 120), 2),
+        (True, (-2, 2), (-120, 110), 2),
+        (True, (-2, 2), (-.001, 0.002), -3),
+        (True, (0, 0), (-1e5, 1e5), 5),
+        (True, (6, 6), (-1e5, 1e5), 6),
+    ]
```

Comment:
can you add a scalar example into here?

Reply:
Since we've decided not to take in scalar anymore, this is a moot point..., but just for posterity sake, my intention was to allow scalar input at the ticklabel_format API level, not at this lower level, where the test sits. The scalar input would then be translated to pair of (repeated) values for lower level consumption.